docs/docs/guides/integrate/retrieve-user-roles.md

```diff
@@ -0,0 +1,470 @@
+
+<img
+    src="/docs/img/guides/integrate/retrieve-user-roles-1.png"
+    width="75%"
+    alt="Assert Roles on Authentication"
+/>
+
+If you need them included in your ID Token, select **’User Roles Inside ID Token’** in application settings. This has to be set in your applications as this is dependent on your application type. Navigate to your application and select this setting.
+
+<img
+    src="/docs/img/guides/integrate/retrieve-user-roles-2.png"
+    width="75%"
+    alt="Assert Roles on Authentication"
+/>
+
+
+Alternatively, you can include the claims `urn:iam:org:project:roles` or/and `urn:zitadel:iam:org:projects:roles` in your scope to achieve the same as above. 
+
+### Retrieve roles from the userinfo endpoint
+
+The user info endpoint is  **ZITADEL_DOMAIN/oidc/v1/userinfo**.
+
+This endpoint will return information about the authenticated user.
+Send the access token of the user as `Bearer Token` in the `Authorization` header:
+
+**cURL Request:**
+```bash
+curl --request GET \
+ --url $ZITADEL_DOMAIN/oidc/v1/userinfo
+ --header 'Authorization: Bearer <TOKEN>'
+```
+
+If the access token is valid, the information about the user (depending on the granted scopes) is returned. Check the [Claims page](https://zitadel.com/docs/apis/openidoauth/claims) for more details. 
+
+**Sample responses:**
+
+**1. Scope used:** `openid email profile urn:zitadel:iam:org:project:id:zitadel:aud` 
+
+**Sample response**: 
+
+```bash
+{
+  "email": "david.wallace@dundermifflin.com",
+  "email_verified": true,
+  "family_name": "Wallace",
+  "gender": "male",
+  "given_name": "David",
+  "locale": "en",
+  "name": "David Wallace",
+  "nickname": "David",
+  "preferred_username": "david.wallace",
+  "sub": "223427827918176513",
+  "updated_at": 1689669364,
+  "urn:zitadel:iam:org:project:223281986649719041:roles": {
+    "cfo": {
+      "223281939119866113": "corporate.user-authorizations-io8epz.zitadel.cloud"
+    },
+    "corporate member": {
+      "223279178798072065": "org-a.user-authorizations-io8epz.zitadel.cloud",
+      "223279223391912193": "org-b.user-authorizations-io8epz.zitadel.cloud"
+    }
+  },
+  "urn:zitadel:iam:org:project:roles": {
+    "cfo": {
+      "223281939119866113": "corporate.user-authorizations-io8epz.zitadel.cloud"
+    },
+    "corporate member": {
+      "223279178798072065": "org-a.user-authorizations-io8epz.zitadel.cloud",
+      "223279223391912193": "org-b.user-authorizations-io8epz.zitadel.cloud"
+    }
+  }
+}
+```
+
+This request can be tested out in the following way:
+1. Select the **‘Assert Roles on Authentication’** checkbox.
+2. Do not include the roles claims in the scope.
+3. When you run the command, you will see that the roles were returned.
+4. If you unselect the **‘Assert Roles on Authentication’** checkbox, you will not see the roles.
+
+
+**2. Scope used:** `openid email profile urn:zitadel:iam:org:project:id:223281986649719041:aud urn:iam:org:project:roles urn:zitadel:iam:org:projects:roles`
+
```

Comment:
I would add a note that they should use the roles from "urn:zitadel:iam:org:project:223281986649719041:roles" iinstead of "urn:zitadel:iam:org:project:roles":
we do have both for backwards compatibility, but the one with the id is the new one

Reply:
Done